contributed_definitions/nyaml/NXenergydispersion.yaml

```diff
@@ -40,18 +40,78 @@ NXenergydispersion(NXobject):
+          Here, WF is the spectrometer work function.
+          This mode is often used in XPS/UPS because the energy resolution does not change with
+          changing energy (due to the constant pass energy).
+
+          Synonyms: constant ΔE mode, constant analyser energy mode, CAE mode, FAT mode
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 12.64
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.64
+      fixed_retardation_ratio:
+        doc:
+        - |
+          constant ΔE/E mode, where the pass energy is scanned such that the electron retardation
+          ratio is constant. In this mode, electrons of all energies are decelerated with this same
+          fixed factor. Thus, the pass energy is proportional to the kinetic energy. This mode is often
+          used in Auger electron spectroscopy (AES) to improve S/N for high-KE electrons, but this
+          leads to a changing energy resolution (∆E ~ Ep) at different kinetic energies. It can however
+          also be used in XPS.
+
+          Synonyms: constant ΔE/E mode, constant retardation ratio mode, CRR mode, FRR mode
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 12.66
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.66
+      fixed_energy:
+        doc: |
+          In the fixed energy (FE) mode, the intensity for one single kinetic energy is measured for a
+          specified time. This mode is particulary useful during setup or alignment of the
+          electron analyzer, for analysis of stability of the excitation source or for sample
+          alignment.
+
+          Since the mode measures intensity as a function of time, the difference in channel signals 
+          is not of interest. Threfore, the signals from all channels are summed.
+
+          Synonyms: FE mode
+      snapshot:
+        doc: |
+          Snapshot mode does not involve an energy scan and instead collects data from all channels of
+          the detector without averaging. The resulting spectrum reflects the energy distribution of
+          particles passing through the analyzer using the current settings. This mode is commonly used
+          to position the detection energy at the peak of a peak and record the signal, enabling faster
+          data acquisition within a limited energy range compared to FAT. Snapshot measurements are
+          particularly suitable for CCD and DLD detectors, which have multiple channels and can accurately
+          display the peak shape. While five or nine-channel detectors can also be used for snapshot
+          measurements, their energy resolution is relatively lower.
+      transmission:
+        doc:
+        - |
+          MISSING DOC.
+        - |
+          xref:
+            spec: <spec>
+            term: <term>
+            url: <url>
```

Comment:
This produces a warning during documentation build and breaks the building of the process. Maybe without `<` and `>`?

Reply:
Error:

```
Warning, treated as error:
/Users/domna/fairmat/nexus_definitions/build/manual/source/classes/contributed_definitions/NXenergydispersion.rst:112:Unknown target name: "<term> <<url>>".
make[1]: *** [html] Error 2
make: *** [local] Error 2
```

Reply:
removed